### ISSUE TITLE
KHPP Beset kiosk login trouble

### ISSUE THREAD

**Issue by lbestock** [urapadmin/kiosk#699]
I am reporting a ticket for someone else, so I refuse to take responsibility for accurately reporting actions elsewhere. I report what I was told and what I did.

Leslie could not log in to Beset KHPP with her old login of law and password. I reset the password  and told her to try again.

She was not able to log in with law and no password, either.

I created a user test, named Leslie, with no password. She was able to get in using this. Then I thought, wait a second I should see if I can get in with law no password since I said "reset password":

![image](https://user-images.githubusercontent.com/49291909/87075089-d0a36300-c1ed-11ea-8060-f04ae7845a7e.png)

I cannot.

Then I thought I would reset my own password and see if that also meant I couldn't get back in but that seems dumb, I will not do that because what if Leslie needs someone else in there right now? And anyway this should be enough for you to figure out the problem, and it is yours to figure out.

**Comment by urapadmin** [comment]
ok, there is no bug, just some lack of information here that amounted to what look like chaotic behaviour on the server's side:
  - law has forgotten her password and used the wrong one OR when I transfered the law database I transfered a password that was not what she thought was the latest. I see no evidence for technical issues in the sever logs. It just says "wrong password" or "wrong user" over and over again.
  - setting the checkbox "reset password" does not reset the password. It means: When the user logs in next, force him or her to change the password. But only after the correct, current password was entered. Since that password was unknown for law, this did not work. The option does not empty the password.
  - The correct way to give a user a new, temporary password is this:
![image](https://user-images.githubusercontent.com/38838314/87148624-065c5080-c2af-11ea-8ba3-b84f2c44e954.png)
If you press that button (ja, it's a button) a toast opens, telling you the new temporary password for the user and - if I am not mistaken, that would be worth a test - after the user typed it in the system will ask for a new password automatically.
